pymc3/stats.py

```diff
@@ -964,3 +964,33 @@ def bfmi(trace):
     energy = trace['energy']
 
     return np.square(np.diff(energy)).mean() / np.var(energy)
+
+
+def r2_score(y_true, y_pred):
```

Comment:
What about rounding the results by default (maybe 2 decimals) and adding an argument to control this...